fitbenchmarking/results_processing/tables.py

```diff
@@ -232,6 +243,18 @@ def enable_link(result):
         # Update table values to point to individual support pages
         table.applymap(enable_link)
 
+        # add performance profile information
+        if name[0] in 'acc':
```

Comment:
In general I'd use `==` instead of `in` here, unless there's something I've missed.

Reply:
Changed -- was a hangover from when I was checking for the name in a list